ui/components/Flex.tsx

```diff
@@ -22,6 +22,10 @@ const Styled = (component) => styled(component)`
   ${({ wide }) => wide && "width: 100%"};
   ${({ wrap }) => wrap && "flex-wrap: wrap"};
   ${({ end }) => end && "justify-content: flex-end"};
+  &.application-actions {
```

Comment:
Similar to my comment about the `<Button />`, the `<Flex />` component doesn't know about `applications`. It exists in a chaotic universe where there are only misaligned `divs` that need to be evenly distributed :wink:.

Reply:
Looking at it again, and yes it is a bad and ugly class that will be destroyed 🔥